test/e2e/clusterlogging_test.go

```diff
@@ -121,8 +121,9 @@ func clusterLoggingInitialDeploymentTest(t *testing.T, f *framework.Framework, c
 			LogStore: logging.LogStoreSpec{
 				Type: logging.LogStoreTypeElasticsearch,
 				ElasticsearchSpec: logging.ElasticsearchSpec{
-					NodeCount: 1,
-					Resources: esResources,
+					NodeCount:        1,
+					Resources:        esResources,
+					RedundancyPolicy: elasticsearch.ZeroRedundancy,
```

Comment:
this change looks unrelated?

Reply:
It is not.  There actually is an OLM validation failure that doesn't manifest itself now in our e2e test because we don't go through OLM.